CHANGELOG.md

```diff
@@ -51,6 +54,7 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 
 [Unreleased]: https://github.com/giantswarm/kubectl-gs/compare/v0.5.3...HEAD
+[0.6.0]: https://github.com/giantswarm/kubectl-gs/compare/v0.5.3...v0.6.0
```

Comment:
No need for this line, it will be added by automation

```suggestion
[0.6.0]: https://github.com/giantswarm/kubectl-gs/compare/v0.5.3...v0.6.0
```

Reply:
Fixed!